src/Plane/PlaneFileGlue.hpp

```diff
@@ -37,5 +38,7 @@ namespace PlaneGlue
 /**
  * Throws exception on error.
  */
-void WriteFile(const Plane &plane, Path path);
+  void WriteFile(const Plane &plane, Path path);
```

Comment:
Why change indentation?

Reply:
aligned to the scope of this function with one indent